File: java/src/main/java/itp/itp1_5/ITP1_5_C.java
package itp.itp1_5;

import java.io.PrintWriter;
import java.util.Scanner;
import java.util.stream.IntStream;

public class ITP1_5_C {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    PrintWriter pw = new PrintWriter(System.out);

    while (true) {
      int h = sc.nextInt();
      int w = sc.nextInt();

      if (h == 0 && w == 0) break;

      IntStream.range(0, h).forEach(i -> {
        IntStream.range(0, w).forEach(j -> {
          pw.print((i + j) % 2 == 0 ? "#" : ".");
        });
        pw.println();
      });
      pw.println();
    }
    pw.flush();
  }
}

File: kotlin/src/itp/itp1_5/ITP1_5_C.kt
package itp.itp1_5

fun main(args: Array<String>) {
    while (true) {
        val (H, W) = readLine()!!.split(" ").map(String::toInt)
        if (H == 0 && W == 0) break
        (1..H).forEach { h ->
            (1..W).forEach { w ->
                print(if (h%2 xor w%2 == 0) "#" else ".")
            }
            println()
        }
        println()
    }
}